sdk/instrumentation/net/http/handler.go

```diff
@@ -17,21 +18,27 @@ type handler struct {
 	defaultAttributes        map[string]string
 	spanFromContextRetriever sdk.SpanFromContext
 	dataCaptureConfig        *config.DataCapture
+	requestFilter            filter.Filter
```

Comment:
should this just be called `filter`, as the same filter will potentially handle both request and response data.

Reply:
done